docs/Installation.md

```diff
@@ -591,7 +583,7 @@ path:
 
 ```sh
 sed -i '' '/[[:<:]]engine[[:>:]]/ s/=.*/= SQLite/' /usr/local/etc/zonemaster/backend_config.ini
-sed -i '' '/[[:<:]]database_name[[:>:]]/ s:=.*:= /var/db/zonemaster/db.sqlite:' /usr/local/etc/zonemaster/backend_config.ini
+sed -i '' '/[[:<:]]file[[:>:]]/ s:=.*:= /var/db/zonemaster/db.sqlite:' /usr/local/etc/zonemaster/backend_config.ini

```

Comment:
For this replacement to work, there cannot be any other key "file", which is not unlikely since that is a generic term. Maybe "databasefile" would be better?

Reply:
Apparently SQLite can use more than one file. https://sqlite.org/fileformat.html
I'll rename it to database_file.